stac_mjx/main.py

```diff
@@ -84,9 +86,9 @@ def run_stac(cfg: DictConfig):
         utils.save(fit_data, fit_path)
 
     # Stop here if skipping transform
-    if cfg.test.skip_transform == 1:
+    if cfg.skip_transform == 1:
         logging.info("skipping transform()")
-        return fit_path, "none"
+        return fit_path, "No transform path"
```

Comment:
```suggestion
        return fit_path, None
```

Checking for `NoneType` is a bit more idiomatic than string comparison.